src/sentry/issues/grouptype.py

```diff
@@ -502,6 +502,18 @@ class PerformanceStreamedSpansGroupTypeExperimental(GroupType):
     default_priority = PriorityLevel.LOW
 
 
+@dataclass(frozen=True)
+class DBInjectionVulnerabilityGroupType(GroupType):
+    type_id = 1020
+    slug = "db_injection_vulnerability"
+    description = "Potential Database Injection Vulnerability"
```

Comment:
```suggestion
    description = "Potential Database Query Injection Vulnerability"
```
somewhat a nit but the injection is happening in the query itself rather than the database